wiki/planning/non-a-star-planning.md

```diff
@@ -0,0 +1,115 @@
+---
+date: {}
+title: Extensions To A* for Dynamic Planning
+published: true
+---
+## Introduction
+This wiki aims to build on the already excellent A* implementation guide by providing information about different planning approaches to counter the pitfalls of standard A* when applied to physical robots. In the scope of this article are two popular extensions - Theta Star and Safe Interval Path Planning (SIPP) that each have their strengths and weaknesses.
```

Comment:
Would be great to link to the [page on motion planning and A-star](https://roboticsknowledgebase.com/wiki/planning/planning-overview/#a) from the site.